### PR TITLE
Document basic JSON-RPC api in CLI options

### PR DESCRIPTION
**Pull request by ann0see**
**Short description of changes**
Adds the jsonrpc command line options.

**Context: Fixes an issue? Related issues**

Related to: #717 (at least a first step)

**Status of this Pull Request**
Ready for review

**What is missing until this pull request can be merged?**
Finished.

**Does this need translation?**
YES.

## Checklist

- [x] I've verified that this Pull Request follows the [general code principles](https://github.com/jamulussoftware/jamulus/blob/master/CONTRIBUTING.md#jamulus-projectsource-code-general-principles)
- [ ] I waited some time after this Pull Request was opened and all GitHub checks completed without errors. 
- [x] I'm sure that this Pull Request goes to the correct branch

### CODE REVIEW

**Comment by ann0see** [comment]
@dtinth could you please quickly review this?

File: wiki/en/Include-Shared-Commands.md [hunk 1]
```diff
@@ -2,10 +2,13 @@
 [comment]: # (This is an include file for use in multiple documents)
 
 - `-h` or `--help`     Display help text         
-- `-i` or `--inifile`   Set location of initialization file (overrides default) 
+- `-i` or `--inifile`   Set location of initialization file (overrides default)
```
Comment (pljones) [inline wiki/en/Include-Shared-Commands.md]: Spacing?

Comment (pljones) [reply wiki/en/Include-Shared-Commands.md]: And line 6?

File: wiki/en/Include-Shared-Commands.md [hunk 2]
```diff
@@ -2,10 +2,13 @@
...
-- `-p` or `--port`    Sets the local UDP port number. Default is 22124 
+- `-p` or `--port`    Sets the local UDP port number. Default is 22124
+- `--jsonrpcport`     Enables JSON-RPC API server to control the app, set TCP port number (EXPERIMENTAL, APIs might change; only accessible from localhost). Please see [the JSON-RPC API Documentation file](https://github.com/jamulussoftware/jamulus/blob/master/docs/JSON-RPC.md).
+- `--jsonrpcsecretfile` Required when using `--jsonrpcport`. Sets a path to a text file containing an authentication string for getting access to the JSON-RPC API.
+
 - `-Q` or `--qos`     Sets the quality of service DSCP value. Default is 128. Disable with 0. QoS is ignored by Windows. To enable it, [see this page](QOS-Windows)
 - `-t` or `--notranslation`   Disable UI language translations
 -  `-6` or `--enableipv6`      Enable IPv6 addressing (IPv4 is always enabled)
-- `-v` or `--version`   Output version information and exit 
+- `-v` or `--version`   Output version information and exit
```
Comment (pljones) [inline wiki/en/Include-Shared-Commands.md]: Spacing?

File: wiki/en/Include-Shared-Commands.md (hunk 1 above)
Comment (ann0see) [reply wiki/en/Include-Shared-Commands.md]: Spacing is wrong in the whole file, I think?

File: wiki/en/Include-Shared-Commands.md (hunk 2 above)
Comment (ann0see) [reply wiki/en/Include-Shared-Commands.md]: Please see my new push

File: wiki/en/Include-Shared-Commands.md (hunk 1 above)
Comment (pljones) [reply wiki/en/Include-Shared-Commands.md]: I guess the spaces get collapsed, anyway.  And `.md` and tables don't always get on...

**Review by pljones — APPROVED** [review]
At least it's more visually appealing :).

File: wiki/en/Include-Shared-Commands.md [hunk 3]
```diff
@@ -1,11 +1,13 @@
...
-- `-v` or `--version`   Output version information and exit 
+- `-h` or `--help`           Display help text         
+- `-i` or `--inifile`        Set location of initialization file (overrides default)
+- `-n` or `--nogui`          Disable GUI (for use in headless mode)                      
+- `-p` or `--port`           Sets the local UDP port number. Default is 22124
+- `--jsonrpcport`            Enables JSON-RPC API server to control the app, set TCP port number (EXPERIMENTAL, APIs might change; only accessible from localhost). Please see [the JSON-RPC API Documentation file](https://github.com/jamulussoftware/jamulus/blob/master/docs/JSON-RPC.md).
+- `--jsonrpcsecretfile`      Required when using `--jsonrpcport`. Sets a path to a text file containing an authentication string for getting access to the JSON-RPC API.
+- `-Q` or `--qos`            Sets the quality of service DSCP value. Default is 128. Disable with 0. QoS is ignored by Windows. To enable it, [see this page](QOS-Windows)
+- `-t` or `--notranslation`  Disable UI language translations
+-  `-6` or `--enableipv6`    Enable IPv6 addressing (IPv4 is always enabled)
```
Comment (dtinth) [inline wiki/en/Include-Shared-Commands.md]: Extra space here.

```suggestion
- `-6` or `--enableipv6`     Enable IPv6 addressing (IPv4 is always enabled)
```

**Review by dtinth — APPROVED** [review]
Yup, looks great!

File: wiki/en/Include-Shared-Commands.md (hunk 1 above)
Comment (ann0see) [reply wiki/en/Include-Shared-Commands.md]: MD tables should work?

**Comment by gilgongo** [comment]
Don't forget to add this to the man page.

EDIT: I don't know how to ~~I can~~ add it to https://github.com/jamulussoftware/jamulus/pull/2448

**Pull request merged**